message_sender/views.py

```diff
@@ -230,6 +230,18 @@ def fire_delivery_hook(outbound):
         )
 
 
+def decr_message_count(message):
+    if not message.channel:
+        channel = Channel.objects.get(default=True)
+    else:
+        channel = Channel.objects.get(
+            channel_id=message.channel)
```

Comment:
I think `message.channel` should give you a channel object, so we don't need to fetch it here.

We can also do a `select_related` when we fetch the message above, so that it doesn't result in 2 database requests.